### PR TITLE
fix(Widget): add new prop to extend the headerActions prop functional in compact type

### PR DESCRIPTION
**Pull request by hamikhambardzumyan**

### CODE REVIEW

**Review by NairaMisakyan — DISMISSED** [review]

File: stories/molecules/Widget/Widget.stories.jsx [hunk 1]
```diff
@@ -34,6 +34,7 @@ export default {
         comparisonText: args({ control: 'text', ...content }),
         withShadow: args({ control: 'boolean', ...appearance }),
         withBorder: args({ control: 'boolean', ...appearance }),
+        withHeaderActions: args({ control: 'boolean', ...appearance }),
```
Comment (GaroGabrielyan) [inline stories/molecules/Widget/Widget.stories.jsx:37]: By adding the `withHeaderActions` prop, we adding additional functionality so please remove this prop from appearance to functionality group.

**Review by GaroGabrielyan — CHANGES_REQUESTED** [review]

File: src/lib/molecules/Widget/index.js [hunk 2]
```diff
@@ -91,7 +90,7 @@ function Widget({
                 </>
             ) : (
                 <>
-                    <li className="image">{img && <img src={img} alt="" />}</li>
+                    {img && <li className="image">{img && <img src={img} alt="" />}</li>}
```
Comment (GaroGabrielyan) [inline src/lib/molecules/Widget/index.js]: Please remove redundant second `img` check.
```suggestion
                    {img && <li className="image"><img src={img} alt="" /></li>}
```

File: src/lib/molecules/Widget/index.scss [hunk 3]
```diff
@@ -44,7 +44,7 @@
         grid-template-areas: 'image head' 'image text';
 
         &.t-default {
-            @include rtl(padding, 0 rem(20) 0 0, 0 0 0 rem(20));
+            @include rtl(padding, 0 rem(20), 0 rem(20));
```
Comment (GaroGabrielyan) [inline src/lib/molecules/Widget/index.scss:47]: We can change this line with:
```suggestion
         padding-inline: rem(20);
```

**Review by GaroGabrielyan — CHANGES_REQUESTED** [review]

**Review by GaroGabrielyan — DISMISSED** [review]

**Review by NairaMisakyan — CHANGES_REQUESTED** [review]
If the widget size value is big and the img prop is present, the image aligns with the border.
image (13)

**Review by NairaMisakyan — APPROVED** [review]

**Pull request merged**